src/DOMWorld.ts

```diff
@@ -615,20 +506,13 @@ class WaitTask {
     this._cleanup();
   }
 
-  _cleanup() {
+  _cleanup(): void {
     clearTimeout(this._timeoutTimer);
     this._domWorld._waitTasks.delete(this);
-    this._runningTask = null;
```

Comment:
this was never used or read anywhere across the entire codebase so I removed this line

Reply:
good catch!